mathics_django/web/controllers/doc.py

```diff
@@ -21,12 +22,37 @@
 
 DocResponse = Union[HttpResponse, JsonResponse]
 
+seen_pymathics_modules = copy(pymathics_modules)
+
 
 def check_for_pymathics_load():
+    global seen_pymathics_modules
     if seen_pymathics_modules != pymathics_modules:
-        # print("XXX refresh pymathics doc")
-        global documentation
-        documentation = MathicsDjangoDocumentation()
+        print("XXX refresh pymathics doc", pymathics_modules)
```

Comment:
This is now uncommented?

Reply:
Sorry about this. I forgot to comment it out again.